src/compilers.c

```diff
@@ -819,6 +819,11 @@ compiler_gcc_args_lto(void)
 static const struct args *
 compiler_cl_args_compile_only(void)
 {
+	/*
```

Comment:
They are here to have a link for the reference implementation in meson. In case I disappear, another guy will be able to take over the windows port easily with them, I think.